content/blog/2020/07/2020-07-30-github-checks-api-plugin-coding-phase-2.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "GitHub Checks API Plugin Project - Coding Phase 2"
+tags:
+- github
+- api
+- plugins
+- developer
+- gsoc
+- gsoc2020
+author: XiongKezhi
+description: "Google Summer of Code 2020 - GitHub Checks API Plugin project update for Coding Phase 2"
+opengraph:
```

Comment:
Would be better to have a custom image. I can help with a draft

Reply:
that would be nice, thanks.

Reply:
https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.g8f0f0276ed_0_0 maybe?